errai-jaxrs/errai-jaxrs-client/src/test/java/org/jboss/errai/enterprise/jaxrs/client/test/QueryParamIntegrationTest.java

```diff
@@ -96,6 +97,13 @@ public void testGetWithQueryParamSetOfStrings() {
         new SimpleAssertionCallback<>("@GET with Set<String> as @QueryParams failed", strings)).getWithQueryParamSetOfStrings(strings);
   }
 
+  @Test
+  public void testGetWithQueryParamSetOfStringsTreeSet() {
+    final Set<String> strings = new TreeSet<>(Arrays.asList("1", "2", "3"));
+    call(QueryParamTestService.class,
+            new SimpleAssertionCallback<>("@GET with Set<String> as @QueryParams failed", strings)).getWithQueryParamSetOfStringsTreeSet(strings);
+  }
+
```

Comment:
why is this here in alignment PR?

Reply:
brought in by the PR @tiagobento raised to fix failing tests in errai